Sources/TuistCore/Models/Target.swift

```diff
@@ -115,10 +115,12 @@ public struct Target: Equatable, Hashable {
             let base = AbsolutePath(sourcePath.dirname)
 
             // Paths that should be excluded from sources
-            let excluded = source.excluding
-                .map { AbsolutePath($0) }
-                .map { excludePath in AbsolutePath(excludePath.dirname).glob(excludePath.basename) }
-                ?? []
+            var excluded: [AbsolutePath] = []
+            source.excluding?.forEach { path in
+                let absolute = AbsolutePath(path)
+                let globs = AbsolutePath(absolute.dirname).glob(absolute.basename)
```

Comment:
Nitpick: you could use `flatMap` and simply return the array of paths created here

Reply:
I think that `reduce` would suit better, because for every item of the `excluding` array another array of `globs` is generated, which needs to be appended to the `excluded` array.

As a newcomer to the project it took my a while to understand what each type in the transformation is, so I decided to use the code as you see, without flat mapping - reduce  magic 🔮.